README.md

```diff
@@ -54,29 +54,25 @@ The web service has the following features
    ```
    https://github.com/fedora-infra/mote
    ```
-2. Download the archive of meeting logs from the link below.
-   ```
-   https://mega.nz/file/cJYykbKA#jJozcnIG-WzwlYVQUXF25lqM5A8PNl2knQObQrSpOSk
-   ```
-3. Clone the forked repository to your local storage.
+2. Clone the forked repository to your local storage.
    ```
    $ git clone git@github.com:<username>/mote.git
    ```
-4. Navigate into the cloned repository.
+3. Navigate into the cloned repository.
    ```
    $ cd mote
    ```
-5. Depending on the kind of approach required, follow either of the below sections.
+4. Depending on the kind of approach required, follow either of the below sections.
 
 ### Containerized setup
 
 1. Install [Podman](https://podman.io/getting-started/installation) on Fedora Linux.
    ```
    $ sudo dnf install podman
    ```
-2. Extract the previously downloaded archive of meeting logs in the directory of the cloned repository.
+2. Download meeting logs in the directory of the cloned repository.
    ```
-   $ tar -xzf meetbot.tar.gz
+   $ wget -r -P "./meetbot" -nH -R "index.html,robots.txt" -A ".html,.txt" https://meetbot-raw.fedoraproject.org/
```

Comment:
Could you please also add that the said URL can be modified depending on which channels logs and minutes are needed like you did for the `fedora-meeting` example?